api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -299,4 +299,13 @@ message CommonTlsContext {
   // Custom TLS handshaker. If empty, defaults to native TLS handshaking
   // behavior.
   config.core.v3.TypedExtensionConfig custom_handshaker = 13;
+
+  // The path to save the TLS key log. The default value is /var/log/envoy/tlskey.log
+  string tls_keylog_path = 15;
```

Comment:
I suggest moving these three fields into their own message and then having an instance of that message here.  That will make it cleaner to add new fields related to key logging here in the future.

Reply:
It make sense. I will change it.